dist/rpm/openQA.spec

```diff
@@ -554,6 +555,10 @@ fi
 %config %{_sysconfdir}/apache2/vhosts.d/openqa.conf.template
 %config %{_sysconfdir}/apache2/vhosts.d/openqa-common.inc
 %config %{_sysconfdir}/apache2/vhosts.d/openqa-ssl.conf.template
+# nginx vhost
+%dir %{_sysconfdir}/nginx
+%dir %{_sysconfdir}/nginx/vhosts.d
+%config %{_sysconfdir}/nginx/vhosts.d/openqa.conf
```

Comment:
This broke openqa.opensuse.org with

```
Jul 02 03:30:43 ariel systemd[1]: Starting The nginx HTTP and reverse proxy server...
Jul 02 03:30:43 ariel nginx[1806]: nginx: [emerg] duplicate upstream "webui" in /etc/nginx/vhosts.d/openqa.conf:3
Jul 02 03:30:43 ariel nginx[1806]: nginx: configuration file /etc/nginx/nginx.conf test failed
```

And when removing the file conf.d/openqa.conf this yields 403 Forbidden. For now I reverted to the old file conf.d and removed the vhosts file. See my comment in https://progress.opensuse.org/issues/131024#note-11 as well